tools/BUILD.bazel

```diff
@@ -13,6 +13,8 @@ nodejs_binary(
         "@npm//@bazel/typescript",
     ],
     entry_point = "@npm//:node_modules/@bazel/typescript/internal/tsc_wrapped/tsc_wrapped.js",
+    # TODO: turn on --worker_sandboxing and remove this flag to see failure to load the plugin
```

Comment:
Is this a TODO for us or just a hint?

Reply:
To compare with components or rules_nodejs